### ISSUE TITLE
unbreakable shuriken isn't unbreakable

### ISSUE THREAD

**Issue by mangogamr** [GTNewHorizons/NewHorizons#5681]
#### Which modpack version are you using?
2.0.8.3
#
#### If in multiplayer; On which server does this happen? 
modded minecraft club
#
#### What did you try to do, and what did you expect to happen?
make an unbreakable shuriken (obsidian part and 7 obsidian plates making unbreaking 10/unbreakable)
make an unbreakable shuriken
#
#### What happened instead? (Attach screenshots if needed)
its not unbreakable
https://cdn.discordapp.com/attachments/683423411875217460/684463584990068751/unknown.png
#
#### What do you suggest instead/what changes do you propose?
idk what to put here? but um, would be nice to have an actually unbreakable shuriken (may also be bugged for the bows and cross bows and all those other projectile based tinkers/if there are any others)

**Comment by mitchej123** [comment]
Pretty sure that's intended - It's not breaking, it's depleting ammo

https://github.com/GTNewHorizons/TinkersConstruct/blob/master/src/main/java/tconstruct/library/weaponry/AmmoItem.java#L62-L68

**Comment by mangogamr** [comment]
it depletes ammo and hits a mob, but i get the shuriken back then right? well i dont
i lose the ammo

**Comment by mangogamr** [comment]
> Pretty sure that's intended - It's not breaking, it's depleting ammo
> 
> https://github.com/GTNewHorizons/TinkersConstruct/blob/master/src/main/java/tconstruct/library/weaponry/AmmoItem.java#L62-L68

it depletes ammo and hits a mob, but i get the shuriken back then right? well i dont
i lose the ammo

**Comment by mitchej123** [comment]
Unbreaking doesn't mean infinite ammo.

Add moss to it if you want it to magically repair.

**Comment by mangogamr** [comment]
what does it mean then? whats the use of adding unbreaking

**Comment by mitchej123** [comment]
It's a modifier that means nothing on that tool/weapon.  Not all modifiers apply to all tools.

**Comment by mangogamr** [comment]
in other mod packs and stuff ive seen and made unbreakable shurikens

**Comment by mitchej123** [comment]
I'm assuming 1.12? You can't compare later versions to 1.7.10.

**Comment by mangogamr** [comment]
idk, well thats bs, alright theough, thanks for looking. wish it blocked us from putting the modifier on if its useless though, have a nice day i guess

**Comment by mitchej123** [comment]
Standard behavior for 1.7.10 tinkers afaik.  We haven't changed it, and not aware of any intent to

**Comment by Banderi** [comment]
I know this is intended behavior/not a bug, although if I could suggest a small thing, would it be possible to mention it on the _"Reinforced X? Ubreakable!"_ quest line text? (similar to how it already says upgrading to Reinfoced X is not often worth it)
The unbreakable/infinite TConstruct shuriken is very a popular combo and commonly used by players in many modpacks, so seeing no warning many people (me included) would try it out of habit only to realize they just wasted an enormous amount of resources and time. I had to try searching online if I was doing something wrong, or if there was a config setting that explicitly disabled the behavior in the pack, but I couldn't find any information whatsoever until I searched Github.
